src/reporter/coverage-report.xsl

```diff
@@ -46,6 +46,35 @@
    <xsl:variable name="xspec-doc" as="document-node(element(x:description))"
       select="doc($xspec-uri)" />
 
+   <!-- Get the *-result.xml file. It contains the date/time the test was run. -->
+   <xsl:variable name="result-uri" as="xs:string"
+      select="replace(base-uri(), '-coverage\.xml', '-result.xml')" />
+   <xsl:variable name="result-doc" as="document-node(element(x:report))">
+      <xsl:choose>
+         <xsl:when test="doc-available($result-uri)">
+            <xsl:sequence select="doc($result-uri)"/>
+         </xsl:when>
+         <xsl:otherwise>
+            <!-- End-to-end coverage tests don't have access to *-result.xml.
```

Comment:
I believe a standard end-user situation would reach `xsl:when` because the test result report is created immediately before the coverage report. Even if the user has customized the paths to the HTML reports, the two XML files should be consistent with the definition of `result-uri`. However, this repo's end-to-end testing does some things differently, so I had to add the `xsl:otherwise` element.